sprint3/controllers/articleController.js

```diff
@@ -0,0 +1,67 @@
+import { PrismaClient } from "@prisma/client";
+import { PatchArticle } from '../structs.js';
+import { assert } from "superstruct";
+
+const prisma = new PrismaClient();
+
```

Comment:
method 별로 기능과 입출력 파라미터에 대한 주석이 있으면 더욱 좋습니다.
ex)

```suggestion
/**
* 상품 목록 조회
* @param (offset, limit)
* @return products
*/
```